1st-gen/tools/styles/README.md

```diff
@@ -1,124 +1,241 @@
+@import '@spectrum-web-components/styles/tokens-v2/global-vars.css';
 ```
 
-This file brings together the global variables and font settings with the "Lightest" color set and "Medium" scale system specification.
+#### Spectrum tokens
 
-```
-@import '@spectrum-web-components/styles/all-large-darkest.css';
-```
+Spectrum uses a base + override pattern. Import base tokens first, then system-specific overrides:
 
-This file brings together the global variables and font settings with the "Darkest" color set and "Large" scale system specification.
+```css
+/* import base tokens */
+@import '@spectrum-web-components/styles/tokens/light-vars.css';
+@import '@spectrum-web-components/styles/tokens/medium-vars.css';
+@import '@spectrum-web-components/styles/tokens/global-vars.css';
 
-```
-@import '@spectrum-web-components/styles/all-large-dark.css';
+/* import spectrum-specific overrides */
+@import '@spectrum-web-components/styles/tokens/spectrum/light-vars.css';
+@import '@spectrum-web-components/styles/tokens/spectrum/medium-vars.css';
+@import '@spectrum-web-components/styles/tokens/spectrum/global-vars.css';
 ```
 
-This file brings together the global variables and font settings with the "Dark" color set and "Large" scale system specification.
+#### Express tokens
 
-```
-@import '@spectrum-web-components/styles/all-large-light.css';
-```
+Express also uses a base + override pattern (same as Spectrum above, but with `express/` paths):
 
-This file brings together the global variables and font settings with the "Light" color set and "Large" scale system specification.
+```css
+/* import base tokens */
+@import '@spectrum-web-components/styles/tokens/light-vars.css';
+@import '@spectrum-web-components/styles/tokens/medium-vars.css';
+@import '@spectrum-web-components/styles/tokens/global-vars.css';
 
-```
-@import '@spectrum-web-components/styles/all-large-lightest.css';
+/* import express-specific overrides */
+@import '@spectrum-web-components/styles/tokens/express/light-vars.css';
+@import '@spectrum-web-components/styles/tokens/express/medium-vars.css';
+@import '@spectrum-web-components/styles/tokens/express/global-vars.css';
 ```
 
-This file brings together the global variables and font settings with the "Lightest" color set and "Large" scale system specification.
+#### Tokens usage
 
-## Color sets
+Which tokens files you import will depend on which tokens you want to use.
 
-### Darkest
+##### V2 tokens
 
-```
-@import '@spectrum-web-components/styles/theme-darkest.css';
-```
+Generally speaking, `light-vars` and `dark-vars` files contain raw color custom property definitions (such as `--spectrum-red-500`), as well as semantic custom property definitions that change depending on which color option you use. For instance, in the v2-tokens, `--spectrum-neutral-subdued-background-color-default` uses different values per theme. It resolves to `--spectrum-gray-700` for light and `--spectrum-gray-500` for dark.
```

Comment:
Perhaps instead of `v2-tokens`, we use `tokens-v2` since that's the directory we have? 

I'll leave that up to you- not a blocker.